crates/bevy_scene/src/scene.rs

```diff
@@ -61,6 +62,33 @@ impl Scene {
         };
 
         let type_registry = type_registry.read();
+
+        // Resources archetype
+        for (component_id, _) in self.world.storages().resources.iter() {
+            let component_info = self
+                .world
+                .components()
+                .get_info(component_id)
+                .expect("component_ids in archetypes should have ComponentInfo");
+
+            let type_id = component_info
+                .type_id()
+                .expect("Reflected resources must have a type_id");
```

Comment:
Nit: For consistency
```suggestion
                .expect("reflected resources must have a type_id");
```